src/common/src/error.rs

```diff
@@ -123,6 +123,11 @@ pub enum ErrorCode {
     #[error("Error while interact with meta service: {0}")]
     MetaError(String),
 
+    /// This error occurs when the meta node receives heartbeat from a previous removed worker
+    /// node. Currently we don't support re-register, and the worker node need a full restart.
+    #[error("Unknown worker")]
```

Comment:
Why this error isn't part of `MetaError`?

Reply:
+1 for putting it into the `MetaError` variant.

Reply:
Because actually I wanted to write something like `matches!(err.inner(), ErrorCode::UnknownWorker)` at client side, and didn't want to `err.to_string().contains("balabala")`.

But I found client side will only get `ErrorCode::Internal` instead of other variants, so it has to do a string match? And then it became the current status (I thought `contains(&ErrorCode::UnknownWorker.to_string())` is at least slightly better than the literal). 

Want some suggestions to make it better. 🤪